File: VB/DemoWebSite/Controllers/InvoiceController.vb
'========================================================================
' This conversion was produced by the Free Edition of
' Instant VB courtesy of Tangible Software Solutions.
' Order the Premium Edition at https://www.tangiblesoftwaresolutions.com
'========================================================================

Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.Mvc
Imports DemoWebSite.Filters

Namespace DemoWebSite.Controllers
	Public Class InvoiceController
		Inherits Controller

		' GET: Invoice
		<BasicAuthenticationFilter("testUser", "testPassword")>
		Public Function Index() As ActionResult
			Return View()
		End Function
	End Class
End Namespace

File: CSharp/DemoWebSite/Controllers/InvoiceController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;
using DemoWebSite.Filters;

namespace DemoWebSite.Controllers
{
    public class InvoiceController : Controller
    {
        // GET: Invoice
        [BasicAuthenticationFilter("testUser", "testPassword")]
        public ActionResult Index()
        {
            return View();
        }
    }
}